XiEditor/Search.swift

```diff
@@ -163,17 +163,18 @@ class FindViewController: NSViewController, NSSearchFieldDelegate, NSControlText
 
 extension EditViewController {
     func openFind(replaceHidden: Bool) {
+        let offset = findViewController.view.fittingSize.height
+
         if findViewController.view.isHidden {
             findViewController.view.isHidden = false
+            let origin = scrollView.contentView.visibleRect.origin
+            scrollView.contentView.scroll(to: NSMakePoint(origin.x ,origin.y - offset))
```

Comment:
I don't think I understand this change. Was there a problem you were trying to solve?

This now creates a new issue if you open the 'replace' view while the find view is open; we don't correctly adjust to the height of the replace view.